### DISCUSSION TITLE
MacOS - no more. I get it. Apple doesnt make it easy.

### DISCUSSION THREAD

**Discussion by eddyfree** [jamulussoftware/jamulus#3217] [General]
That’s my take on it anyway.  Is that accurate? 

I'm a multi-instrumentalist & audio engineer with well over three decades of experience. 
I was THRILLED to find Jamulus and spent considerable time reading everything here;

https://jamulus.io/

I was getting more excited by the minute.   Sadly,  after coming here I find that MacOS is no longer supported.   PLEASE UPDATE the supported platforms at the above website to convey current information.  It will no doubt be greatly appreciated by many.

Is iOS still an option, albeit experimental, or is it off the table as well? 

I may be willing to invest in a PC to do this, depending on what other software I find.  I have never participated in Mac vs PC vs Linux arguments.  To be honest,  FOSS appeals much more to me in principle than its corporate competitors. It's simply that nearly every professional studio I've ever worked in (that used computers) were running OSX.  For efficiency and workflow I followed suit. 

I will continue searching for other options. Do you have any suggestions that you could recommend to me?

Keep up the good work!
ed

**Comment by chrisrimple** [comment]
MacOS is still supported by Jamulus.  You can download the latest installer (for recent MacOS versions) from http://www.jamulus.io.  There is also a build for older MacOS versions available from the site.

**Comment by mcfnord** [comment]
I've heard for Jamulus Client on Mac there are visual quirks, but the audio works 100%.

I did my first year on a Raspberry Pi 4. A little unstable, but certainly affordable.

I believe an outsider does build an iOS version that is widely used.

**Reply by chrisrimple** [reply]
I've been running Jamulus on MacOS (older and newer) since May 2020.  Any visual quirks have long-since been resolved and it works great.

**Reply by eddyfree** [reply]
Thank you for all the replies.  I must have misread.

**Comment by jujudusud** [comment]
First page:
![Screenshot_2024-01-01-13-07-13-950_org mozilla firefox-edit](https://github.com/jamulussoftware/jamulus/assets/12748768/ef6ae0b1-6efa-4688-bfca-ad54eb0b09c2)

**Comment by hoffie** [comment]
> Sadly, after coming here I find that MacOS is no longer supported.

What made you think so? Is there some specific URL which implies that or some specific experience during your attempt to run it?

**Reply by gilgongo** [reply]
I'd like to know that too. The only thing I can think of is that becuase we list Windows first on the list of platforms on the home page, and in their excitement they might have skipped straight to "other platforms" and then didn't see the .dmgs there?

**Reply by ann0see** [reply]
The issue is that we don't have a signing cert anymore. Therefore macOS throws an error.

**Reply by gilgongo** [reply]
$99 USD per year, I think.

**Reply by ann0see** [reply]
Yes. @danryu signs koord and should have one. So maybe we can set it up with him on his repo

**Reply by danryu** [reply]
Apologies for the slow response.
Yes this is definitely a possibility. No guarantees but I'm hoping to be able to action this in the next 2-4 weeks.
